df_py/predictoor/models.py

```diff
@@ -71,14 +70,72 @@ def accuracy(self):
         return self._accuracy
 
 
+class PredictionSummary:
+    @enforce_types
+    def __init__(self, prediction_count, correct_prediction_count, contract_addr):
+        self.prediction_count = prediction_count
+        self.correct_prediction_count = correct_prediction_count
+        self.contract_addr = contract_addr
+
+    @property
+    def accuracy(self) -> float:
+        if self.prediction_count == 0:
+            return 0
+        return self.correct_prediction_count / self.prediction_count
```

Comment:
you can also do a one-liner: `return self.correct_prediction_count / self.prediction_count if self.prediction_count else 0`